Directory.Build.props

```diff
@@ -28,7 +28,11 @@
     <RootNamespace>GraphQL.Server.$(MSBuildProjectName)</RootNamespace>
     <PackageId>GraphQL.Server.$(MSBuildProjectName)</PackageId>
 
-    <GraphQLVersion>6.0.0-preview-612</GraphQLVersion>
+    <GraphQLVersion>7.0.0-preview-641</GraphQLVersion>
+
+    <SignAssembly>true</SignAssembly>
+    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
```

Comment:
```suggestion
```

https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#generateassemblyinfo

I would remove this property from other projects too.